### PR TITLE
Cleaning up UI strings.

### PR DESCRIPTION
**Pull request by pniko**

### CODE REVIEW

File: src/extension/appcenter/appCenterStrings.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ export class ACStrings {
     public static FetchDeploymentsStatusBarMessage: string = 'Fetching app deployments for you...';
     public static GettingAppInfoMessage: string = 'Getting app info...';
     public static DetectingAppVersionMessage: string = 'Detecting app version...';
-    public static RunningReactNativeBundleCommandMessage: string = 'Running \"react-native bundle\" command...';
+    public static RunningBundleCommandMessage: string = 'Running bundle command...';
```
Comment (pniko) [inline src/extension/appcenter/appCenterStrings.ts]: @max-mironov please review the changes.

Comment (max-mironov) [reply src/extension/appcenter/appCenterStrings.ts]: @pniko need to replace this string https://github.com/Microsoft/vscode-cordova/blob/master/src/extension/appcenter/command/commandExecutor.ts#L245 then, no probs, I'll take care of it in separate commit!

**Comment by max-mironov** [comment]
@pniko Im merging this for Cordova extension, do we need to update the corresponding strings for ReactNative extension?

**Comment by sergey-akhalkov** [comment]
@max-mironov, my understanding is that `appcenter` folder contains almost the same code for both extensions, so I believe that this folder should be as generic as possible to make it simpler to synchronize and maintain it in the future.

**Comment by max-mironov** [comment]
Ya, in this case not generic but probably almost the same (except cordova/react specific parts), I'll take care of it!

**Pull request merged**